src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -1298,6 +1301,33 @@ public static void createBucketWithLifeCycle() throws IOException, URISyntaxExce
     gcsTargetBucketName = createGCSBucketLifeCycle();
     BeforeActions.scenario.write("GCS target bucket name - " + gcsTargetBucketName); }
 
+  @Before(order = 1, value = "@DATASTORE_SOURCE_ENTITY")
+  public static void createEntityInCloudDataStore() throws IOException, URISyntaxException {
+    kindName = "cdf-test-" + UUID.randomUUID().toString().substring(0, 8);
+   String entityName = DataStoreClient.createKind(kindName);
+    PluginPropertyUtils.addPluginProp("kindName", entityName);
+    BeforeActions.scenario.write("Kind name - " + entityName + "created successfully");
```

Comment:
`" created successfully"`

similar comment applies to whole file

Reply:
done.